sklearn/linear_model/tests/test_base.py

```diff
@@ -153,6 +154,31 @@ def test_linear_regression_sparse(random_state=0):
         assert_array_almost_equal(ols.predict(X) - y.ravel(), 0)
 
 
+@pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
```

Comment:
This is the only case, you will not need it (you are catching it with pytest in the test itself :))